store/app/api/db.py

```diff
@@ -33,19 +33,26 @@ async def create_tables(crud: Crud | None = None) -> None:
         await asyncio.gather(
             crud._create_dynamodb_table(
                 name="Users",
-                columns=[
-                    ("email", "S", "HASH"),
-                    # ("banned", "B", "RANGE"),
-                    # ("deleted", "B", "RANGE"),
+                keys=[
+                    ("id", "S", "HASH"),
+                ],
+                gsis=[
+                    ("emailIndex", "email", "S", "HASH"),
                 ],
             ),
             crud._create_dynamodb_table(
                 name="Tokens",
-                columns=[
-                    ("email", "S", "HASH"),
-                    # ("issued", "N", "RANGE"),
-                    # ("disabled", "B", "RANGE"),
+                keys=[
+                    ("id", "S", "HASH"),
+                ],
+                gsis=[("emailIndex", "email", "S", "HASH")],
```

Comment:
nit

```suggestion
                gsis=[
                    ("emailIndex", "email", "S", "HASH"),
                ],
```

reason is that if we add more indexes it keeps the file blame easier to parse

Reply:
I initially did that, `make format` is what made it all in one line. I concur, I would have written it in multiple lines, but probably best to obey the linter.

Reply:
if you put a comma inside the list, black will format it this way